lighthouse-core/audits/deprecations.js

```diff
@@ -30,10 +31,572 @@ const UIStrings = {
+  'Triggering `window.alert` from cross origin iframes has been deprecated and will be removed in the future.',
+  /**
+  *@description TODO(crbug.com/1318851): Description needed for translation
+  */
+  crossOriginWindowConfirm:
+  'Triggering `window.confirm` from cross origin iframes has been deprecated and will be removed in the future.',
+  /**
+  *@description TODO(crbug.com/1320339): Description needed for translation
+  */
+  cssSelectorInternalMediaControlsOverlayCastButton:
+  'The `disableRemotePlayback` attribute should be used in order to disable the default Cast integration instead of using `-internal-media-controls-overlay-cast-button` selector.',
+  /**
+  *@description TODO(crbug.com/1320340): Description needed for translation
+  */
+  customCursorIntersectsViewport:
+  'Custom cursors with size greater than 32x32 DIP intersecting native UI is deprecated and will be removed.',
+  /**
+  *@description This warning occurs when a script modifies `document.domain`
+  *    without having set on `Origin-Agent-Cluster` http header. In other
+  *    words, when a script relies on the default behaviour of
+  *    `Origin-Agent-Cluster` when setting document.domain.
+  */
+  documentDomainSettingWithoutOriginAgentClusterHeader:
+  'Relaxing the same-origin policy by setting `document.domain` is deprecated, and will be disabled by default. To continue using this feature, please opt-out of origin-keyed agent clusters by sending an `Origin-Agent-Cluster: ?0` header along with the HTTP response for the document and frames. See https://developer.chrome.com/blog/immutable-document-domain/ for more details.',
+  /**
+  *@description Warning displayed to developers when the non-standard `Event.path` API is used to notify them that this API is deprecated.
+  */
+  eventPath: '`Event.path` is deprecated and will be removed. Please use `Event.composedPath()` instead.',
+  /**
+  *@description Warning displayed to developers when the Geolocation API is used from an insecure origin (one that isn't localhost or doesn't use HTTPS) to notify them that this use is no longer supported.
+  */
+  geolocationInsecureOrigin:
+  '`getCurrentPosition()` and `watchPosition()` no longer work on insecure origins. To use this feature, you should consider switching your application to a secure origin, such as HTTPS. See https://goo.gle/chrome-insecure-origins for more details.',
+  /**
+  *@description Warning displayed to developers when the Geolocation API is used from an insecure origin (one that isn't localhost or doesn't use HTTPS) to notify them that this use is deprecated.
+  */
+  geolocationInsecureOriginDeprecatedNotRemoved:
+  '`getCurrentPosition()` and `watchPosition()` are deprecated on insecure origins. To use this feature, you should consider switching your application to a secure origin, such as HTTPS. See https://goo.gle/chrome-insecure-origins for more details.',
+  /**
+  *@description TODO(crbug.com/1318858): Description needed for translation
```

Comment:
Will translators still attempt to translate these strings without a description?

Reply:
When translators ask us why there isn't a context description, are we going to have to tell them that it's it not our fault, we got these from somewhere else.

Or in theory should that never happen, because these specific strings will be referred to the existing translations from DT.